package.json

```diff
@@ -32,11 +32,11 @@
     "uuid": "^3.3.2"
   },
   "devDependencies": {
```

Comment:
`azure-pipelines-task-lib` should be updated to the `preview` version as well

https://github.com/microsoft/azure-pipelines-tool-lib/blob/f507fdcd6d7d2f46472a951c85277cb7deb48ed4/package.json#L25-L33

Reply:
I will update when `azure-pipelines-task-lib` is released